multiplayer/src/epics/pauseGameAsync.ts

```diff
@@ -0,0 +1,18 @@
+import { state, dispatch } from "../state";
+import * as gameClient from "../services/gameClient";
+import { showModal } from "../state/actions";
+
+export async function pauseGameAsync() {
+    const { clientRole } = state;
+
+    try {
+        pxt.tickEvent("mp.pausegame", { role: clientRole! });
+        if (clientRole === "host") {
+            await gameClient.pauseGameAsync();
+            // TODO: pause simulator
```

Comment:
@jwunderl what message should be sent here to pause the sim? And similarly in `resumeGameAsync.ts`.

Reply:
Should be these guys https://github.com/microsoft/pxt/blob/master/multiplayer/src/components/ArcadeSimulator.tsx#L50-L56, I had left them in for testing but they're currently unused

Reply:
And worth a note, should only really need to pause host, as the clients will just 'pause' by virtue of the screen not receiving updates / buttons doing nothing

Reply:
And the simulator render in grayscale when paused, like we do in the editor? Is that just a css effect?

Reply:
Oh we can do this on on the sim container. Easy.

Reply:
Right, if you want to match what it looks like in editor it should be whatever applies when you do simdriver().stop() - we don't really use a 'pause' state in the editor outside of debug mode so just need to dupe it here